.circleci/config.yml

```diff
@@ -513,17 +560,260 @@ workflows:
+            - connector-base
+      - connector:
+          name: mqtt
+          artifact: connector-mqtt
+          requires:
+            - connector-base
+      - connector:
+          name: rest-swagger
+          artifact: connector-rest-swagger
+          requires:
+            - connector-base
+      - connector:
+          name: salesforce
+          artifact: connector-salesforce
+          requires:
+            - connector-base
+      - connector:
+          name: sftp
+          artifact: connector-sftp
+          requires:
+            - connector-base
+      - connector:
+          name: slack
+          artifact: connector-slack
+          requires:
+            - connector-base
+      - connector:
+          name: sql
+          artifact: connector-sql
+          requires:
+            - connector-base
+      - connector:
+          name: telegram
+          artifact: connector-telegram
+          requires:
+            - connector-base
+      - connector:
+          name: timer
+          artifact: connector-timer
+          requires:
+            - connector-base
+      - connector:
+          name: twitter
+          artifact: connector-twitter
+          requires:
+            - connector-base
+      - connector:
+          name: servicenow
+          artifact: connector-servicenow
+          requires:
+            - connector-base
+      - connector:
+          name: webhook
+          artifact: connector-webhook
+          requires:
+            - connector-base
+      - connector:
+          name: concur
+          artifact: connector-concur
+          requires:
+            - connector-base
+      - connector:
+          name: kudu
+          artifact: connector-kudu
+          requires:
+            - connector-base
+      - connector:
+          name: odata
+          artifact: connector-odata
+          requires:
+            - connector-base
+      - connector:
+          name: knative
+          artifact: connector-knative
+          requires:
+            - connector-base
+      - connector:
+          name: flow
+          artifact: connector-flow
+          requires:
+            - connector-base
       - meta:
           requires:
-            - connector
+            - activemq
+            - apiprovider
+            - amqp
+            - awss3
+            - awssqs
+            - awssns
+            - box
+            - dropbox
+            - email
+            - ftp
+            - fhir
+            - gmail
+            - googlecalendar
+            - googlesheets
+            - http
+            - irc
+            - jira
+            - kafka
+            - log
+            - mqtt
+            - restswagger
+            - salesforce
+            - sftp
+            - slack
+            - sql
+            - telegram
+            - timer
+            - twitter
+            - servicenow
+            - webhook
+            - concur
+            - kudu
+            - odata
+            - knative
+            - flow
```

Comment:
This is a bit ugly to see... too much duplication. Isn't there any way to group those and reference with a unique label instead?

Reply:
agreed, but yet I have not found a better way to do it

Reply:
Maybe someone with experience on the tool can tell (@zregvart ?)